numpy/core/src/umath/ufunc_object.c

```diff
@@ -2089,6 +2107,124 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
         goto fail;
     }
 
+    if (axis) {
+        /*
+         * possibly remap axes.
+         */
+        int axis_rqst;
```

Comment:
I may be a little influenced by having to follow [this](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules) in my daily work, but wouldn't things be more readable if this was `requested_axis`, or `axis_for_all`, or something more descriptive?